quizroyale/server/tokenizer.py

```diff
@@ -0,0 +1,16 @@
+import datetime
```

Comment:
`tokenizer` is a component used in lexical analysis of code:

https://en.wikipedia.org/wiki/Lexical_analysis

How about naming it `token_factory`?